x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/ilm/IndexLifecycleService.java

```diff
@@ -423,6 +423,7 @@ PolicyStepsRegistry getPolicyRegistry() {
     }
 
     static Set<String> indicesOnShuttingDownNodesInDangerousStep(ClusterState state, String nodeId) {
+        // GWB-> Handle REPLACE here
```

Comment:
I think this branch still needs to be handled since we would exit early on accident without handling it